UefiCpuPkg/PiSmmCpuDxeSmm/SmmProfile.c

```diff
@@ -1345,18 +1320,17 @@ SmmProfilePFHandler (
     //
     // If SMM profile does not start, call original page fault handler.
     //
-    SmiDefaultPFHandler ();
+    SmiProfileMapPFAddress ();

```

Comment:
please consider whether need call this function if SMM profile does not start since we create the full page table.

Reply:
Added a new commit to remove it

Reply:
ok